src/services/schedule.ts

```diff
@@ -30,6 +30,19 @@ type DbSchedule = {
   AllCourses: DbCourse[];
 };
 
+function detectMobile() {
+  if (window.innerWidth >= 1514) {
+    return false;
+  }
+}
+
+function mobileFormat() {
+  if (detectMobile() == false) {
+    return ' | ';
+  } else {
+    return ' ';
+  }
+}
```

Comment:
You only use these functions once and I'm not sure if 1514px equates to mobile. Seems oddly wide.
```suggestion
function mobileFormat() {
  return window.innerWidth >= 1514 ? ' | ' : '';
}
```